hypervisor/Cargo.toml

```diff
@@ -8,7 +8,7 @@ version = "0.1.0"
 [features]
 kvm = ["kvm-bindings", "kvm-ioctls", "vfio-ioctls/kvm"]
 mshv = ["mshv-bindings", "mshv-ioctls", "mshv_emulator", "vfio-ioctls/mshv"]
-mshv_emulator = ["iced-x86"]
+mshv_emulator = ["iced-x86", "mshv-bindings"]
```

Comment:
Let's keep changes to the "hypervisor" crate to the same commit, including the ones from "hypervisor/src/lib.rs".